dev/nvim-lsp.nix

```
@@ -80,6 +80,9 @@ let
       settings = {
         ['nil'] = {
           testSetting = 42,
+          formatting = {
+              command = { "nixpkgs-fmt" },
```

Indentation is 2 here.
```suggestion
            command = { "nixpkgs-fmt" },
```